UnitTests/BraintreePayPalTests/BTPayPalDriver_Checkout_Tests.swift

```diff
@@ -552,6 +552,30 @@ class BTPayPalDriver_Checkout_Tests: XCTestCase {
         XCTAssertTrue(postedAnalyticsEvents.contains("ios.paypal-single-payment.webswitch.credit.offered.started"))
     }
 
```

Comment:
We should probably cover the case for defaulting to NO if `offer_pay_later` isn't set. 

Looks like we do that for `offer_paypal_credit` in `testCheckout_whenRemoteConfigurationFetchSucceeds_postsPaymentResourceWithShippingAddress`.